### ISSUE TITLE
name typo in get_parameter

### ISSUE THREAD

**Issue by zoltuz** [BuildACell/BioCRNPyler#138]
```python
def get_sbml_id(self):
        sbml_id = self.paramter_key.name+"_"
        if self.parameter_key.part_id is not None:
```
paramter_key doesn't exist. 
in parameter.py L170

**Comment by WilliamIX** [comment]
What do you mean? parameter_key is a property of ParameterEntry...

@property
    def parameter_key(self) -> ParameterKey:
        return self._parameter_key

**Comment by zoltuz** [comment]
code has ```sbml_id = self.paramter_key.name+"_"``` instead param**e**ter

**Comment by WilliamIX** [comment]
See slack

**Comment by zoltuz** [comment]
Fixed in PR #134
